### DISCUSSION TITLE
PHP 8 support

### DISCUSSION THREAD

**Discussion by TSchuermans** [rollbar/rollbar-php-laravel#119] [Q&A] [answered]
Is there already a plan to support PHP 8?

**Comment by danielmorell** [comment]
Hey Tom, sorry for the long wait on your question. The short answer is, yes. 

Getting there will be a bit of work, but I am here to make that happen. If you would like to help we would welcome your assistance.

**Comment by ghost** [comment, marked as answer]
@TSchuermans We recently released version [v4.0.0](Rollbar SDK v4 incorporated a major refactor to add better typing and interoperability with the latest packages and standards in the PHP ecosystem that are not supported by PHP 8.2 onwards.). It incorporated a major refactor to add better typing and interoperability with the latest packages and standards in the PHP ecosystem that are not supported by PHP 8.2 onwards.
